common/changes/@gooddata/sdk-ui-all/ch1_2023-09-06-13-15.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@gooddata/sdk-ui-all",
+      "comment": "Added support for pivot tables, that let you move column headers to rows.",
```

Comment:
Added support for pivot tables, allowing you to transpose column headers to rows for more flexible data analysis.